docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+services:
+  db:
+    image: postgres:13
+    container_name: BioCirV_ETL_db
+    env_file:
+      - .env
+    ports:
+      - "${POSTGRES_PORT}:5432"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DB}"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: BioCirV_ETL_app
+    env_file:
+      - .env
+    environment:
+      - PREFECT_API_URL=http://prefect-server:4200/api
+      - PYTHONPATH=/app
+    volumes:
+      - .:/app
+      - /app/.pixi
+    working_dir: /app
+    depends_on:
+      db:
+        condition: service_healthy
+    command: tail -f /dev/null
+
+  prefect-server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: BioCirV_ETL_prefect_server
+    env_file:
+      - .env
+    ports:
+      - "4200:4200"
+    volumes:
+      - .:/app
+      - /app/.pixi
+      - prefectdata:/root/.prefect
+    working_dir: /app
+    depends_on:
+      db:
+        condition: service_healthy
+    command: pixi run prefect server start --host 0.0.0.0
```

Comment:
This command of the `prefect server ...` can probably be turned to a pixi task: https://pixi.sh/latest/workspace/advanced_tasks/